README.md

```diff
@@ -1,29 +1,127 @@
-# Convos - Multiuser chat application
+![Convos](http://convos.by/images/logo.png)
 
+Convos is the simplest way to use IRC. It is always online, and accessible to
+your web browser, both on desktop and mobile. Run in on your home server, or
+cloud service easily. It can be deployed to Docker-based cloud services, or
+you can just run it as a normal Mojolicious application, using any of the
+Deployment Guides.
 
- Convos is a multiuser chat application built with Mojolicious.
+![Screenshot](http://convos.by/images/screenshot.png)
 
- It currently support the IRC protocol, but can be extended to support
- other protocols as well.
+We got a [live demo](http://demo.convos.by) that is free to try.
 
+## Features
 
-# Getting Started 
+#### Always online
 
-First we are going to need to install some dependencies:
+The backend server will keep you logged in and logs all the activity in your
+archive.
 
-```bash
-$ ./script/convos install
-```
+### Archive
 
-Once this is done can start Convos by running one of the commands below.
+All chats will be logged and indexed, which allow you to search in earlier
+conversations.
 
-```bash
-$ ./script/convos daemon;
-$ ./script/convos daemon --listen http://*:3000;
-```
+### Avatars
 
-And connect a browser to localhost:3000.
+The chat contains profile pictures which can be retrieved from Facebook or
+from gravatar.com.  The avatar used when talking with Convos is lovingly
+delivered by [Robohash.org](http://robohash.org/convos.by).
```

Comment:
"one-point-oh" branch does not support avatars. I'm not sure how to make that work nicely. It's a bit annoying when 99% of the users doesn't have a gravatar.

See also https://github.com/Nordaaker/convos/issues/252